dts/bindings/pinctrl/st,stm32-pinctrl.yaml

```diff
@@ -78,6 +86,11 @@ child-binding:
             ... {
                      pinmux = <STM32_PINMUX('A', 9, ANALOG)>;
             };
+               GPIO A9 set as GPIO output high
+            ... {
+                     pinmux = <STM32_PINMUX('A', 9, ANALOG)>;
```

Comment:
keeping `ANALOG` feels confusing here, shouldn't this be `GPIO`?

Reply:
oh, right... Will fix thanks